test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1939,6 +1939,22 @@ public void scans() throws Exception {
     ts.exec("deletetable -f t");
   }
 
+  @Test
+  public void scansWithColon() throws Exception {
+    ts.exec("createtable twithcolontest");
+    ts.exec("insert row c:f cq value");
+    ts.exec("scan -r row -cf c:f", true, "value");
+    String result = ts.exec("scan -b row -cf c:f  -cq cq -e row");
+    assertEquals(2, result.split("\n").length);
+    result = ts.exec("scan -b row -c cf -cf c:f  -cq cq -e row", false);
+    assertTrue(result.contains("mutually exclusive"));
+
+    result = ts.exec("scan -b row -cq col1 -e row", false);
+    assertTrue(result.contains("cannot be empty"));
+
```

Comment:
```suggestion
    ts.exec("scan -b row -cf c:f  -cq cq -e row", true, "value");
    ts.exec("scan -b row -c cf -cf c:f  -cq cq -e row", false, "mutually exclusive");
    ts.exec("scan -b row -cq col1 -e row", false, "cannot be empty");
```
A slight simplification so we don't have to create the results object to compare against.